test/update_results_tests.jl

```diff
@@ -132,6 +132,39 @@ subfolders = true, special_list=special_list, black_list = black_list)
 @test sort(names(cres3)) == sort(names(cres2))
 @test size(cres3) == size(cres2)
 
+###############################################################################
+#                           test updating feature                             #
+###############################################################################
+
+folder = datadir("update_test")
+mkdir(folder)
```

Comment:
I'm not a big fan of these persistent files / folders.

This works just fine in CI (because tests only run once) but when testing locally, everything breaks once you rerun individual tests.

@Datseris What is your opinion on this?

Reply:
I agree. I went with what was done in other tests in the same file, but it probably would be better to use temporary directories with some cleanup code that is run regardless if tests fail. I can look into it.

Reply:
Ok, I've pushed a commit that changes the tests to run in a temp directory.